### PR TITLE
rename V2 to main in docs and pipelines

### PR DESCRIPTION
**Pull request by navalev**

### CODE REVIEW

**Review by shiranr — APPROVED** [review]

File: presidio-analyzer/README.MD [hunk 1]
```diff
@@ -16,7 +16,7 @@ Named Entity Recognition and other types of logic to detect PII in unstructured
 
 Use the following button to deploy presidio analyzer to your Azure subscription.
 
-[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fpresidio%2FV2%2Fpresidio-analyzer%2Fdeploytoazure.json)
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fpresidio%2Fmain%2Fpresidio-analyzer%2Fdeploytoazure.json)
 
 TODO: change this link to main branch once merged (#2765).
```
Comment (itye-msft) [inline presidio-analyzer/README.MD:21]:
```suggestion
```

File: presidio-anonymizer/README.MD [hunk 2]
```diff
@@ -9,31 +9,33 @@ entities with desired values.
 
 Use the following button to deploy presidio anonymizer to your Azure subscription.
 
-[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fpresidio%2FV2%2Fpresidio-anonymizer%2Fdeploytoazure.json)
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fpresidio%2Fmain%2Fpresidio-anonymizer%2Fdeploytoazure.json)
 
 TODO: change this link to main branch once merged (#2765).
```
Comment (itye-msft) [inline presidio-anonymizer/README.MD:14]:
```suggestion
```

**Review by SharonHart — APPROVED** [review]

**Review by itye-msft — CHANGES_REQUESTED** [review]
I believe few comments should be removed.
I was not sure why the json in the readme had to change in this PR

File: presidio-analyzer/README.MD (hunk 1 above)
Comment (navalev) [reply presidio-analyzer/README.MD:21]: the deploy to azure button links to an ARM template. we need to update the branch to main

Comment (SharonHart) [reply presidio-analyzer/README.MD]: It's already working after the branching, checked it

**Review by itye-msft — APPROVED** [review]

**Pull request merged**